sdk/go/auto/local_workspace_test.go

```diff
@@ -772,7 +772,7 @@ func TestUpsertStackRemoteSourceWithSetup(t *testing.T) {
 	}
 	project := workspace.Project{
 		Name: tokens.PackageName(pName),
-		Runtime: workspace.NewProjectRuntimeInfo("go", map[string]interface{}{
+		Runtime: workspace.NewProjectRuntimeInfo("go", nil, map[string]interface{}{
```

Comment:
Having to edit here it seems like its a minor breaking change to the API that users might use?

Reply:
Yeh I have no idea if anyone is calling this. Our "common" code is in a weird place of mostly kinda internal for the cli, but maybe people use it. We never call this with a version, so we could just leave it as is and let it default to nil in the method.